integration-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Start a SPIRE Server and Agent and run the integration tests
+# Only works on Linux.
+
+set -euf -o pipefail
+
+spire_folder="spire-858d04b"
+
+function cleanup() {
+  killall -9 spire-agent || true
+  killall -9 spire-server || true
+  rm -f /tmp/spire-server/private/api.sock
+  rm -f /tmp/spire-agent/public/api.sock
+  rm -rf ${spire_folder}
+}
+
+# Some cleanup: kill spire processes that could have remained from previous run
+cleanup
```

Comment:
This can be avoided by using a `trap` here instead
```suggestion
# Some cleanup: kill spire processes that could have remained from previous run
trap cleanup EXIT
```